costs/costs_route.go

```diff
@@ -191,15 +191,20 @@ func makeElasticSearchRequestAndParseIt(ctx context.Context, parsedParams esQuer
 	)
 	res, err := searchService.Do(ctx)
 	if err != nil {
+		if err.Error() == "elastic: Error 404 (Not Found): no such index [type=index_not_found_exception]" {
+			l.Warning("Query execution failed, ES index does not exists : "+index, nil)
```

Comment:
It would cost you nothing to log the error as part of the log data. Better to log too much than not enough, IMO.